dags/openshift_nightlies/config/benchmarks/medium-control-plane.json

```diff
@@ -25,7 +26,28 @@
                 "CLEANUP_WHEN_FINISH": "true",
                 "CLEANUP": "true",
                 "MUST_GATHER_EACH_TASK": "true",
-				"CLEANUP_TIMEOUT": "1h"
+                "CLEANUP_TIMEOUT": "1h"
+            }
+        },
+        {
+            "name": "node-density-cni",
+            "workload": "kube-burner",
+            "trigger_rule": "all_done",
+            "command": "./run.sh",
+            "env": {
+                "WORKLOAD": "node-density-cni",
+                "PODS_PER_NODE": "45",
+                "NODE_COUNT": "120",
+                "JOB_TIMEOUT": "18000",
+                "QPS": "20",
+                "BURST": "20",
+                "STEP_SIZE": "30s",
```

Comment:
just a small nit, the step size for normal node-density is 2m, do we want to keep it as 30s for this?